zk/datadog_checks/zk/zk.py

```diff
@@ -158,8 +159,8 @@ def check(self, instance):
             raise
         else:
             # Parse the response
-            metrics, new_tags, mode, zk_version = self.parse_stat(stat_out)
 
+            metrics, new_tags, mode, zk_version = self.parse_stat(stat_out)
```

Comment:
Let's not change that, I think it made sense to keep the lines of code next to their comments